src/aap_eda/api/views/external_event_stream.py

```diff
@@ -235,13 +251,17 @@ def post(self, request, *_args, **kwargs):
             logger.warning("Error fetching external secrets %s", str(err))
             return Response(status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
+        event_headers = self._redacted_headers(
```

Comment:
in line 245 `logger.debug("Headers %s", request.headers)`, should we just remove or redact sensitive headers?